src/gc/env/gcenv.ee.h

```diff
@@ -82,6 +79,7 @@ class GCToEEInterface
     static void FreeStringConfigValue(const char* key);
     static bool IsGCThread();
     static bool IsGCSpecialThread();
+    static bool CreateThread(void (*threadStart)(void*), void* arg, bool is_special, const wchar_t* name);
```

Comment:
I would try to avoid naming things special wherever possible. `special` is very cryptic.

What about calling this flag `suspendable`?

(Also, it would be nice to rename IsGCSpecialThread to something more self-descriptive.)

Reply:
will do!